lib/jobs/ipmi-sel-alert-job.js

```diff
@@ -46,7 +46,58 @@ function ipmiSelPollerAlertJobFactory(
     }
     util.inherits(IpmiSelPollerAlertJob, PollerAlertJob);
 
+    IpmiSelPollerAlertJob.prototype._formatSelAlert = function _formatSelAlert(data) {
+        /*
+         * data format:
+           { alerts: [ { data: [Object], matches: [Object], action: 'critical' } ],
+             user: 'admin',
+             host: '192.168.188.91',
+             node: '5858c7f7f6f0ce7d08a7298b',
+             lastPreviouslyReadSelEntry: '0000',
+             lastSelDeleteTimeLastRun: '2017-01-10T01:28:13.000Z',
+             workItemId: '5858c89bca4349fa0409ba8c',
+             pollerName: 'sel' }
+         * expected event format:
+           [{ type: 'polleralert',
+              action: 'sel.updated',
+              typeId: '5858c89bca4349fa0409ba8c',
+              nodeId: '5858c7f7f6f0ce7d08a7298b',
+              severity: 'critical',
+              data: {
+                 { user: 'admin',
+                   host: '192.168.188.91',
+                   alert: { reading: [Object], matches: [Object] }
+                 }}} ]
+         */
+        assert.array(data.alerts, 'alerts');
+
+        var template = {
+            type: 'polleralert',
+            action: 'sel.updated',
+            typeId: data.workItemId,
+            nodeId: data.node,
+            severity: '',
+            data: {
+                user: data.user,
+                host: data.host,
+                alert: {}
+            }
+        };
+
+        return _.transform(data.alerts, function(result, item){
+            var tmp = _.cloneDeep(template);
+
+            tmp.data.alert.matches = _.cloneDeep(item.matches);
+            tmp.data.alert.reading = _.cloneDeep(item.data);
+            tmp.severity = item.action;
```

Comment:
I would suggest to hard code this severity from RackHD aspect, not from BMC info